Description: don't check for machine/cpufunc.h on kFreeBSD archs
 On kFreeBSD archs sys/io.h and machine/cpufunc.h can't be included together.
 They contains conflicting function signatures, thus don't look for the
 latter.
 .
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Last-Update: <2012-06-27>

---

--- vice-2.3.dfsg.orig/configure.ac
+++ vice-2.3.dfsg/configure.ac
@@ -2305,7 +2305,14 @@ if test x"$is_unix" = "xyes" -a x"$is_un
     AC_CHECK_HEADERS(ieee1284.h,,[LIBIEEE1284_HEADERS_PRESENT=no],)
     AC_CHECK_HEADERS(sys/mman.h hw/inout.h,,[QNX_MMAN_HEADERS_PRESENT=no],)
     AC_CHECK_HEADERS(machine/sysarch.h,,[BSD_SET_IOPERM_HEADERS_PRESENT=no],)
-    AC_CHECK_HEADERS(unistd.h sys/io.h machine/pio.h machine/cpufunc.h)
+    AC_CHECK_HEADERS(unistd.h sys/io.h machine/pio.h)
+    case "$host_os" in
+      *kfreebsd*)
+        ;;
+      *)
+        AC_CHECK_HEADERS(machine/cpufunc.h)
+        ;;
+    esac
 
     if test x"$LIBIEEE1284_HEADERS_PRESENT" = "xyes"; then
       AC_CHECK_LIB(ieee1284, ieee1284_find_ports,[LIBS="-lieee1284 $LIBS"; PARSID_SUPPORT=yes],,)
